front-end/packages/robustar/src/components/prediction-viewer/Visualizer.vue

```diff
@@ -242,7 +242,7 @@ export default {
 
     open() {
       this.isActive = true;
-      // this.getVisualizeData();
+      this.getVisualizeData();
```

Comment:
Uncommenting this line seems to cause an issue: now in ImageList page, when you click `PREDICT` on an image, `predict` api gets called twice, which results in accessing a locked model (the first call locks up the model). You can check this by opening the `Network` panel of the browser console.

Actually, not only the `predict` API, but also all the other APIs in the visualizer are called twice in ImageList page (but annotation page works fine).